apps/verify/models/write-cookie.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const dataService = require('../../common/data-service');
+const {
+  makeRequest,
+  requestHeaders,
+  } = dataService;
```

Comment:
You don't consume `dataService` directly, so why bother declaring an entire constant?
```suggestion
const {
  makeRequest,
  requestHeaders,
} = require('../../common/data-service');
```

Reply:
thanks, I'll just copy and paste so it's in one commit